### ISSUE TITLE
Have you ever updated your genmon container?

### ISSUE THREAD

**Issue by FlaMike** [JMVS/genmon-docker#1]
Several  months ago we corresponded on @gregmac's thread & you helped me get genmon running in a container.  I've never updated my container & wondered if you ever have done so.  As was  the case when I installed the container originally, there does not appear to be a genmon_ubuntu repository that's  readily accessible.  I don't know if the repository has been kept up to date or not, either.

If you have updated your container, could you kindly tell me how you did it?

Many thanks for your help!

**Comment by JMVS** [comment]
Hello. Yeah, I remember. It seems you're mixing some conecepts here, like I did when I started. Let me try to clarify:
**genmon_ubuntu**: This is a **local** image, meaning that is created only on your machine and does not exists outside that realm _(that is, unless someone creates one and makes it public)_.
This local image is created by the **Dockerfile** file which in turn pull the **latest ubuntu image** _(which does exist in the Docker Hub)_ and then incorporates the genmon software.
This means that if you want to "update" the genmon_ubuntu image all you have to do is recreate the image with the Dockerfile, providing you're preserving genmon's config files in a permanent storage outside the image or that you created a manual backup from inside genmon and restore it afterwards.

And to answer you question, no, I did not update the genmon_ubuntu base image as I didn't feel the need, just genmon with it self-updating process.

Hope this helps!

**Comment by FlaMike** [comment]
Thank you for your helpful response.  I apologize for my lack of understanding but I think I'm on the right path now.

I had not updated the docker image, genmon_ubuntu.  I also did not realize I could update genmon itself outside of the docker universe.  So, I navigated to the genmon UI, went to about, & hit the "Update to latest version" button.  Poof!  I'm on v1.18.18, the current version.

Thanks again for your help & patience!
